repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/examples/container.cfg.xml

```diff
@@ -32,4 +32,10 @@
 -->
         </ssl-configuration>
     </deployment-config>
+
+    <cluster-config cluster-id="internal-cluster">
```

Comment:
Indeed.

Oh, and since this is a node service, the system model is guaranteed to have been read at least once already. After all, that's how we determined the cluster ID and node ID properties that are accessible to all node services. Since the cluster ID is wired in, and we subscribe to the container on `PostConstruct`, we should be guaranteed that the ordering there is always correct.